packages/@tinacms/cli/src/cmds/init/setup-files/index.ts

```diff
@@ -135,12 +133,9 @@ export const nextPostPage =
             <h1 className="text-3xl m-8 text-center leading-8 font-extrabold tracking-tight text-gray-900 sm:text-4xl">
               {data.getPostsDocument.data.title}
             </h1>
-            {/* Convert markdown to html in the browser only */}
-            {typeof window !== "undefined" && (
-              <ContentSection
-                content={window.marked.parse(data.getPostsDocument.data.body)}
```

Comment:
Somewhat uncertain about the removal of this. But this is what I needed to do to make it work. 

Let me know if this is fine.